examples/path_whitelist/README.md

```diff
@@ -0,0 +1,34 @@
+Gohan path whitelist example
+--------------------
+
+In this example, we show how to define a list of paths that never
+require an authorization. A single path is defined as a regular
+expression (regex).
+
+In order to define a whitelisted path, add the following entry
+in the configuration file:
+
+``` yaml
+# path whitelist configuration
+path_whitelist:
```

Comment:
How about creating a route section (which covers all settings related to paths) and add a property called: noauth_paths ?